src/pages/terms.mdx

```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
+
+12.10. **Severability**. If any provision of this Agreement is held illegal or unenforceable by any court of competent jurisdiction, such provision shall be deemed severed from the remaining provisions of this Agreement and shall not affect or impair the validity or enforceability of the remaining provisions of this Agreement. The Parties hereto agree to replace any such illegal or unenforceable provision with a new provision that has the most nearly similar permissible economic or other effect.
+
+12.11. **Controlling Law**. This Agreement will be governed by the ReductStore of the State of Delaware, without reference to conflict of ReductStore principles. In any dispute arising out of this Agreement, Customer and ReductStore each consent to the exclusive personal jurisdiction and venue in the state and federal courts within Middletown, Delaware. The Parties exclude the provisions of the United Nations Convention on Contracts for the International Sale of Goods from this Agreement and any transactions that may be implemented in connection with this Agreement.
 
 ---
 
 # Support Services Addendum
 
-This Support Services Addendum (this **"Addendum"**) describes the Support Services to be provided under the Agreement
-between Customer and ReductStore. Capitalized terms not defined in this Addendum have the meanings given to them in
-the Agreement.
+This Support Services Addendum (this **"Addendum"**) describes the Support Services to be provided under the Agreement between Customer and ReductStore. Capitalized terms not defined in this Addendum have the meanings given to them in the Agreement.
+
+## 1. Definitions
+
+1.1. **Problem** means a failure of the Supported Software to substantially conform to the functional specifications set forth in the applicable documentation provided by ReductStore.
+
+1.2. **Support Services** means each of the services made available by us under this Agreement including those offered via Architecture Review, software support and problem resolution as part of the Service offerings.
 
-## <a id="addendum-1-definitions"></a>1. Definitions
+1.3. **Updates and Upgrades** means subsequently released versions of the Supported Software. For clarity, Updates and Upgrades do not include any products that ReductStore licenses separately from the Supported Software or for an additional fee.
 
-1.1. **Problem** means a failure of the Supported Software to substantially conform to the functional specifications
-set forth in the applicable documentation provided by ReductStore.
+1.4. **Workaround** means a modification or "patch" for a particular version of the Supported Software, which may be of a temporary or interim nature, to help avoid, correct or circumvent a Problem.
```

Comment:
Workaround is not used anywhere else. We could stick with "Problem Resolution".

Reply:
Yes, but it is a part of problem resolution, so let's keep it.